index.js

```diff
@@ -34,7 +34,10 @@ module.exports = function (linkHeader) {
     .map(parseLink)
     .filter(function (x) { return x && x.rel; })
     .reduce(function (acc, x) {
-      acc[x.rel] = x;
+      x.rel.split(/\s+/).forEach(function (rel) {
```

Comment:
I know I'm guilty of this myself, but it'd make things faster to move this function declaration out of the `forEach` and name it.
Not crucial for this PR, but should consider doing this for all the others as well, i.e. `filter, reduce`. Basically the same way it's done for `map` above.